apps/prairielearn/elements/pl-file-editor/pl-file-editor.py

```diff
@@ -168,7 +168,8 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
     # We will store the files in the submitted_answer["_files"] key,
     # so delete the original submitted answer format to avoid
     # duplication
-    del data["submitted_answers"][answer_name]
+    if answer_name in data["submitted_answers"]:
+        del data["submitted_answers"][answer_name]
```

Comment:
this is really a very different type of change than all the rest in this PR, I would rather have it be a separate PR

Reply:
Also, its not clear to me if we want to make this change. Do we care if the element processor crashes if a nonsense POST is made?

Reply:
#11940

Reply:
[4566c96](https://github.com/PrairieLearn/PrairieLearn/pull/11939/commits/4566c9617e5b61451bca259f0e0de66cdaf92354)